dist/appform-field-location.tpl.html.js

```diff
@@ -31,8 +32,13 @@ ngModule.run(['$templateCache', function ($templateCache) {
     '      name="inputNameY"\n' +
     '      ng-model="fieldLocationCtrl.field.value.long"\n' +
     '      ng-required="fieldLocationCtrl.field.required"\n' +
+    '      ng-change="fieldLocationCtrl.updateModel()"\n' +
     '    >\n' +
     '  </md-input-container>\n' +
     '</div>\n' +
+    '\n' +
+    '<div id="error" ng-show="fieldLocationCtrl.field.err">\n' +
+    '  Error: {{fieldLocationCtrl.field.err}}\n' +
+    '</div>\n' +
```

Comment:
It would be nice to standardize error display across the fields.

We need to avoid using `id=error` here because ids should be window-globally unique